### PR TITLE
Fix: Added validation for website URL

### PR DESCRIPTION
**Pull request by tharshita**
# Notion Task

https://www.notion.so/Website-Links-in-templates-dcb668af074f4a86b211ace16fc4dab4

# Changes Made In This PR

- added validation for website URL in contact using regex

# Screenshots

_for UI changes, show a before and after shot_\
**Before:**
no validation
**After:**
Screenshot 2022-04-09 at 12 30 17 PM

Screenshot 2022-04-09 at 12 30 01 PM

### CODE REVIEW

File: src/components/builder/Input.js [hunk 1]
```diff
@@ -133,6 +137,24 @@ const Input = ({
...
+            <input
+              id={uuid}
+              name={name}
+              type={type}
+              value={value}
+              onBlur={onBlur}
+              onChange={onChange}
+              placeholder={placeholder}
+            />
+            {console.log(websiteRegex.test(value), value)}
```
Comment (millasml) [inline src/components/builder/Input.js:151]: remove console.log

File: src/components/builder/left/sections/Contact.js [hunk 2]
```diff
@@ -84,6 +84,7 @@ const Contact = () => {
             <div className={styles.form}>
               <Input
                 name="website"
```
Comment (millasml) [inline src/components/builder/left/sections/Contact.js:86]: i think we can use this type for the linkedin field too

**Review by millasml — CHANGES_REQUESTED** [review]

**Comment by tharshita** [comment]
Okies done!

**Review by millasml — CHANGES_REQUESTED** [review]
one last comment! can you please make it such that the regex accepts empty strings as valid input too?

**Review by millasml — DISMISSED** [review]

**Review by millasml — APPROVED** [review]

**Pull request merged**